cassandra-analytics-core/src/main/java/org/apache/cassandra/bridge/CassandraBridgeFactory.java

```diff
@@ -130,14 +153,20 @@ private static String typesResourceName(@NotNull String label)
         return jarResourceName(label, "types");
     }
 
+    @NotNull
+    private static String sparkSqlResourceName(@NotNull String label)
+    {
+        return jarResourceName(label, "sparksql");
+    }
+
     private static String jarResourceName(String... parts)
     {
         return "/bridges/" + String.join("-", parts) + ".jar";
     }
 
     @NotNull
     @SuppressWarnings("unchecked")
-    private static CassandraBridge create(@NotNull String label)
+    private static Pair<CassandraBridge, SparkSqlTypeConverter> create(@NotNull String label)
```

Comment:
The method signature looks a bit odd. `CassandraBridgeFactory#create` returns a pair.. 

How about create a dedicated data class, say `VersionedCassandraBridge`, that wraps both `CassandraBridge` and `SparkSqlTypeConverter`?  Then `create` and `get` methods return the `VersionedCassandraBridge`. `getSparkSql` may be removed. 

```java
public class VersionedCassandraBridge
{
    public final CassandraBridge cassandraBridge;
    public final SparkSqlTypeConverter sparkSqlTypeConverter;
}
```

Reply:
yeah. I find a proper data class is more readable and maintainable than `Pair` in the long run.